roles/kolibri/files/kolibri.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kolibri server
+
+[Service]
+User=ideascube
+Group=ideascube
+ExecStart=/home/ideascube/.kolibri/env/bin/python2 /home/ideascube/.kolibri/env/bin/kolibri start
```

Comment:
After a few tests, not sure whose the culprit here ansible (py2) or pip (py2, pip3 not installed), but kolibri won't install in his virtualenv. So we'll stick with python2 for now as both are installed on our box and this is not about to change.